.github/.cspell/gamedev_dictionary.txt

```diff
@@ -58,6 +58,7 @@ sublist # any sub-set of elements of a given list
 sublists # plural of sublist
 subrange # a range entirely contained on a given range
 SVGs # plural of SVG
+texturepacker # a packed spritesheet format
```

Comment:
I think this should be on flame_dictionary as it is the name of a tool not a standardad gamedeve term